### ISSUE TITLE
Variable not found on iOS

### ISSUE THREAD

**Issue by richrout** [VitaliiBlagodir/cordova-plugin-datepicker#89]
Hey,

I'm building my application using VS 2015 Tools for Apache Cordova and testing on an iPhone using the ionic application deployment tool.
So far this plugin works on Ripple, Android Emulator and an android phone, but when I try to do test it on the iPhone (using ionic) I am getting a variable not found error. I have tried:
datePicker, window.datePicker, window.plugins.datePicker.
All work on all other emulators and devices, but will always be not found on the iOS (either datePicker undefined, or window.plugins is undefined).

Any ideas? Places to check what I could be doing wrong? It seems I'm the only person with this problem...

**Comment by shrijan** [comment]
Hi,
Did you find any solution to this
It works find with android
datepicker.show(blah,blah,blah);
but gives variable not found error

**Comment by richrout** [comment]
No. I'm not getting any errors on Android. Wasn't able to find a solution
for ios so far...

**Comment by hgo** [comment]
+1

**Comment by astopo** [comment]
- 1, `$window.datePicker` is undefined on iOS 9.2, cordova 5.4.1

**Comment by elliotstokes** [comment]
Anyone looked into/found a solution to this?

**Comment by hieunc229** [comment]
+1

**Comment by hieunc229** [comment]
I found out in my case that I did not include 'cordovar.js' before initiative datePicker function.
